asynction/common_types.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Mapping
+
+JSONMappingValue = Any
+JSONMapping = Mapping[str, JSONMappingValue]
+JSONSchema = JSONMapping
```

Comment:
These could stay in the top level types module. See the comments below with regards to keeping everything under a single types module/package.